### PR TITLE
fix: add thematic break to the Che server parameters reference

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change?

feat: add thematic break to the Che server parameters reference

## What issues does this pull request fix or reference?

readability

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by rkratky — APPROVED** [review]
Lovely!

File: modules/installation-guide/examples/checluster-properties.adoc [hunk 1]
```diff
@@ -118,7 +118,7 @@ identityProviderRoute: Route custom settings.
...
 initialOpenShiftOAuthUser: For operating with the OpenShift OAuth authentication, create a new user account since the kubeadmin can not be used. If the value is true, then a new OpenShift OAuth user will be created for the HTPasswd identity provider. If the value is false and the user has already been created, then it will be removed. If value is an empty, then do nothing. The user's credentials are stored in the `openshift-oauth-user-credentials` secret in 'openshift-config' namespace by Operator. Note that this solution is Openshift 4 platform-specific.
-nativeUserMode: Enables native user mode. Currently works only on OpenShift and DevWorkspace operator. Native User mode uses OpenShift OAuth directly as identity provider, without Keycloak.
+nativeUserMode: Enables native user mode. Currently works only on OpenShift and DevWorkspace engine. Native User mode uses OpenShift OAuth directly as identity provider, without Keycloak.
```
Comment (nickboldt) [inline modules/installation-guide/examples/checluster-properties.adoc:121]: not to nitpick but... how about
```
Currently works only on OpenShift with the DevWorkspace engine enabled.
```
?

**Review by nickboldt — APPROVED** [review]
If Robert approces, so do I. No idea what these UI changes look like though.

**Comment by themr0c** [comment]
> No idea what these UI changes look like though.

I understand that we need better visibility for the build preview.

**Comment by nickboldt** [comment]
So, it's merged but not built?

When will it be pushed to https://www.eclipse.org/che/docs/che-7/installation-guide/advanced-configuration-options-for-the-che-server-component/#che-server-component-system-properties-reference_che ?

**Comment by themr0c** [comment]
> So, it's merged but not built?

Yes, I can merge to `master` late in the night on a smartphone, but to backport to the stable branch I need to boot the laptop the next morning. It's now a matter of minutes.

**Pull request merged**